holmes/plugins/toolsets/json_filter_mixin.py

```
@@ -0,0 +1,126 @@
+        ]
+
+    return value
+
+
+def _apply_jq_filter(data: Any, expression: str) -> Tuple[Optional[Any], Optional[str]]:
+    try:
+        compiled = jq.compile(expression)
+        matches = compiled.input(data).all()
+        if len(matches) == 1:
+            return matches[0], None
+        return matches, None
```

Ruff (0.14.10)

Consider moving this statement to an `else` block

(TRY300)